packages/commandkit/src/app/handlers/AppCommandHandler.ts

```diff
@@ -796,8 +797,8 @@ export class AppCommandHandler {
           data: {
             command: {
               name: command.name,
-              description: `${command.name} command`,
-              type: 1,
+              description: 'No command description set.',
+              type: ApplicationCommandType.ChatInput,
```

Comment:
Everything else looks good honestly, but do we need this `ChatInput` type here? Because this has been defined below under `if (cmd.data.chatInput)` block I think this can be removed. It may have existed because CommandRegistrar did not handle this before